scripts/sync-react.js

```diff
@@ -50,6 +50,7 @@ Or, run this command with no arguments to use the most recently published versio
 `
     )
   }
+  newVersionInfo.releaseLabel = channel
```

Comment:
Otherwise the script made `react-experimental-builtin` point to the Canary release channel.